### ISSUE TITLE
docs: Issue on page Cloud Instance Recommendations for AWS, GCP, and Azure

### ISSUE THREAD

**Issue by nopzdk** [scylladb/scylladb#20121]
I would like to report an issue on page https://enterprise.docs.scylladb.com/branch-2024.1/getting-started/cloud-instance-recommendations

### Problem
in SC we offer n2-highmem-32 with 9000GB, but on this page its only listed for 6000GB

###  Suggest a fix
please correct this disk size if it is incorrect.

**Comment by tzach** [comment]
Thanks @nopzdk good catch

ScyllaDB Cloud do show  9000GB for n2-highmem-32
![image](https://github.com/user-attachments/assets/54d55870-fe07-4f70-ae3e-cb087d61dcbc)

Same problem in open source
https://opensource.docs.scylladb.com/master/getting-started/cloud-instance-recommendations

@nopzdk note you can easily send a PR for such issue 
using the edit this page
![image](https://github.com/user-attachments/assets/c9d2f400-d002-40b6-b484-8769720a2170)

**Comment by nopzdk** [comment]
@tzach I created the PR for opensource, but I do not have that option on Enterprise.

**Comment by tzach** [comment]
> @tzach I created the PR for opensource, but I do not have that option on Enterprise.

Correct. In most cases, open source updates are backported to Enterprise.

**Comment by annastuchlik** [comment]
Fixed with https://github.com/scylladb/scylladb/pull/21537.
Closing.
